grails-app/controllers/org/broadinstitute/orsp/api/IssueListController.groovy

```diff
@@ -0,0 +1,53 @@
+package org.broadinstitute.orsp.api
+
+import grails.converters.JSON
+import grails.rest.Resource
+import groovy.util.logging.Slf4j
+import org.broadinstitute.orsp.AuthenticatedController
+import org.broadinstitute.orsp.Issue
+import org.broadinstitute.orsp.SupplementalRole
+import org.broadinstitute.orsp.utils.IssueUtils
+import org.broadinstitute.orsp.utils.UtilityClass
+
+@Slf4j
+@Resource
+class IssueListController extends AuthenticatedController {
+
+    def getProjectsForUser() {
+        try {
+            new UtilityClass(queryService).registerIssueListMarshaller()
```

Comment:
I missed this in https://github.com/broadinstitute/orsp-pub/pull/216. We should never be writing new constructors like this `new UtilityClass(queryService)`. Instead, `queryService` should be a spring-injected bean like we do in all of our other classes. If a class needs access to a bean, then it is no longer a utility style class and is now something that should be inside the grails world, i.e. a domain class, a service class, a controller class, a job class, etc.